### PR TITLE
support setting dynamic ranges through model inputs

### PR DESCRIPTION
**Pull request by chenbohua3**

### CODE REVIEW

**Review by qiuxiafei — APPROVED** [review]

File: pytorch_blade/tests/test_config.py [hunk 1]
```diff
@@ -162,6 +163,24 @@ def check_invalid(shapes):
             self.assertEqual(now_cfg.dynamic_tuning_shapes[1]['max'], shapes2['max'])
             self.assertEqual(now_cfg.dynamic_tuning_shapes[1]['opts'], shapes2['opts'])
 
+    def test_dynamic_tuning_inputs(self):
```
[the remark this answers is not in the document]
Comment (chenbohua3) [reply pytorch_blade/tests/test_config.py:166]: For some torchscripts exported by torch.jit.script,the format of the input may be very complex, for example, Dict. In this case, it is not easy or even possible to set tuning shapes through List. So this config allows users to directly pass inputs corresponding to min/max/opts

File: pytorch_blade/tests/test_config.py [hunk 2]
```diff
@@ -162,6 +163,24 @@ def check_invalid(shapes):
...
+        min_inp = {1: torch.tensor(1)}
+        max_inp = {3: torch.tensor(3)}
```
Comment (fortianyou) [inline pytorch_blade/tests/test_config.py:168]: Then I think the `1`, and `3`, are not shapes here.

Nit： use `torch.randn([1], device=self.device)` and `torch.randn([3], device=self.device)`

Comment (chenbohua3) [reply pytorch_blade/tests/test_config.py]: device has been added:)

File: pytorch_blade/torch_blade/config.py [hunk 3]
```diff
@@ -150,6 +159,10 @@ def __init__(self):
         # Also, you can pass multiple dynamic input shapes if single configure does not give satisfied
         # performance, like config.dynamic_tuning_shapes = [configure1, configure2, ...].
         self._dynamic_tuning_shapes = {}
+        # For some torchscripts exported by torch.jit.script,the format of the input may be very complex,
+        # for example, Dict. In this case, it is not easy or even possible to set tuning shapes through
+        # List. So this config allows users to directly pass inputs corresponding to min/max/opts
+        self._dynamic_tuning_inputs = {}
```
Comment (fortianyou) [inline pytorch_blade/torch_blade/config.py:165]: Please update the configuration here: https://github.com/alibaba/BladeDISC/blob/main/pytorch_blade/benchmark/detectron2/run_blade.py#L274

Comment (chenbohua3) [reply pytorch_blade/torch_blade/config.py:165]: done

**Review by fortianyou — APPROVED** [review]
LGTM

**Pull request merged**